tests/tabs_mangled.py

```diff
@@ -0,0 +1,7 @@
+class Artist:
```

Comment:
pytest tmpdir instead to keep this inside the test function? you can always write `\t` in the string. just wary tabs might get complained about by future lint rules